packages/lesswrong/server/resolvers/tagHistoryFeed.ts

```diff
@@ -109,6 +123,38 @@ defineFeedResolver<Date>({
             }).fetch();
           },
         } : null),
+        // Summary edits
+        (historyOptions.showSummaryEdits ? {
+          type: "summaryRevision",
+          getSortKey: (rev: DbRevision) => rev.editedAt,
+          doQuery: async (limit: number, cutoff: Date|null): Promise<DbRevision[]> => {
+            return await Revisions.find({
+              documentId: {$in: summaryIds},
+              collectionName: "MultiDocuments",
+              fieldName: "contents",
+              ...(cutoff ? {editedAt: {$lt: cutoff}} : {}),
+            }, {
+              sort: {editedAt: -1},
+              limit,
+            }).fetch();
```

Comment:
This looks like it'll fetch changes to lenses, not just summaries

Reply:
I think the `documentId: {$in: summaryIds}` should sufficiently constraint it?

Reply:
Oh, whoops, yes.